pkg/ocm/ocm.go

```diff
@@ -42,18 +45,21 @@ type OCMInterface interface {
 }
 
 const (
-	ClustersPageSize      = 50
-	ocmNotLoggedInMessage = "Not logged in"
+	ClustersPageSize            = 50
+	ocmNotLoggedInMessage       = "Not logged in"
+	ocmConnectionTimeoutDefault = 30 * time.Second
 )
 
 type DefaultOCMInterfaceImpl struct {
-	//	connection *ocmsdk.Connection
+	Timeout              time.Duration // Timeout for the OCM connection can be set
+	ocmConnectionMutex   sync.Mutex
+	ocmConnectionTimeout *ocmsdk.Connection
```

Comment:
Does this mean `ocmConnectionWithTimeout`?
The name is timeout but the type is connection which is a bit confusing.